ftw/testing/tests/test_freezer.py

```diff
@@ -85,14 +85,20 @@ def test_handles_tzinfo_correctly(self):
             with freeze(dt) as clock:
                 self.assertEquals(dt, datetime.datetime.now(timezone))
                 self.assertEquals(dt_utc, datetime.datetime.utcnow())
+                self.assertEquals(dt, DateTime().asdatetime().astimezone(timezone))
```

Comment:
i think theoretically this should be `normalize`d

```python 
timezone.normalize(DateTime().asdatetime().astimezone(timezone))
```

as the pytz docs states you should do so after performing datetime arithmetics 🤔. On the other hand it does not seem to have an impact here, so up to you for the test.